### ISSUE TITLE
Why did you take the table out?

### ISSUE THREAD

**Issue by syntacticsolutions** [Akryum/vue-virtual-scroller#116]
I was planning on using the table and realized that it was taken out sometime between v0.12.0 and v1.0.0. The only reason I use this thing is to render big data in a virtual scroller table. I'm thinking you should bring it back.

**Comment by Akryum** [comment]
The structure allowing reusing DOM as much as possible for maximum performance isn't compatible with the `<table>` HTML element. It is possible to create tables with it though, with each item being a row of the table and using for example flex to arrange cells horizontally.

**Comment by Aymkdn** [comment]
> The structure allowing reusing DOM as much as possible for maximum performance isn't compatible with the <table> HTML element. It is possible to create tables with it though, with each item being a row <div> of the table and using for example flex to arrange cells horizontally.

Would you mind creating a working example in codepen or somewhere else please ?

**Comment by Aymkdn** [comment]
I did this example that, I believe, is correct: https://codesandbox.io/s/pwlrzmjjk7

**Comment by syntacticsolutions** [comment]
yes but try using it with a sticky header, and side-scroll. It won't work

**Comment by syntacticsolutions** [comment]
It's okay, I made my own virtual scrolling table. No worries

**Comment by sainathsr** [comment]
I have the same problem , I have a table with sticky Header and 1st column , implemented with CSS position stick for first row cells and first column cells . My Table element grows and the table container has a  set height/width and overflow:scroll . I want to now lazy load my rows . Does virtual scroller work  
for my case . ?

**Comment by anhnhoktvn** [comment]
@Akryum could you explain why it's not  compatible with the ```<table>``` HTML element?

**Comment by s6joui** [comment]
I ended up using Quasar's virtual scroll:
https://quasar.dev/vue-components/virtual-scroll#Example--Sticky-headers-table

**Comment by jaybeecave** [comment]
A structured set of divs are never equivlent to a table. 
 
The way tables handle widths and also making sure all columns conform to the data within the table. (I understand this might be an issue when dynamically rendering a set amount of items at a time.

I found a tutorial that put me on to this library, the tut had a table example, it seems like a pretty big breaking change and a shame you wouldn't just keep the old component as a legacy item.

**Comment by m4heshd** [comment]
> It's okay, I made my own virtual scrolling table. No worries

@syntacticsolutions would you care to share your code? I'd love to take a look at it.

**Comment by syntacticsolutions** [comment]
https://www.npmjs.com/package/vue-infinite-scroll-table?activeTab=readme

**Comment by syntacticsolutions** [comment]
> @Akryum could you explain why it's not compatible with the `<table>` HTML element?

It’s because you can’t add fake margin to the top of the inner tbody element. It simply doesn’t work.

**Comment by syntacticsolutions** [comment]
> The structure allowing reusing DOM as much as possible for maximum performance isn't compatible with the `<table>` HTML element. It is possible to create tables with it though, with each item being a row `<div>` of the table and using for example flex to arrange cells horizontally.

You can actually use the `display` css property to arrange the div table cells the same way a table would. 

https://www.npmjs.com/package/vue-infinite-scroll-table?activeTab=readme
